src/components/SignupForm.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+const SignupForm: React.FC = () => {
+    const [email, setEmail] = useState('');
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        //TODO: add signup form logic here(Still setting up accounts)
+        console.log('Email submitted:', email);
```

Comment:
I'd recommend removing console logs 

also instead of having the signupform as stateful you can apply a name field to the input  like "email" and access it via const form = (event.target as HTMLFormElement); and form.elements["email"].value

this allows the component to be stateless

Reply:
also change e to event and type it as React.FormEvent<HTMLFormElement>